Source/WebCore/platform/graphics/cg/GradientRendererCG.cpp

```diff
@@ -33,11 +33,19 @@
 
 namespace WebCore {
 
-GradientRendererCG::GradientRendererCG(ColorInterpolationMethod colorInterpolationMethod, const GradientColorStops& stops)
-    : m_strategy { pickStrategy(colorInterpolationMethod, stops) }
+GradientRendererCG::GradientRendererCG(ColorInterpolationMethod colorInterpolationMethod, const GradientColorStops& stops, std::optional<DestinationColorSpace> destinationColorSpace)
+    : m_strategy { pickStrategy(colorInterpolationMethod, stops, destinationColorSpace) }
 {
 }
 
+std::optional<DestinationColorSpace> GradientRendererCG::colorSpace() const
+{
+    if (std::holds_alternative<Gradient>(m_strategy))
+        return std::get<Gradient>(m_strategy).colorSpace;
```

Comment:
Can't not we use `m_strategy.get_if<Gradient>`? In this case we do not check and get in two separate statements.